### ISSUE TITLE
Support for CMD + ESC, `, . or ~ for a Keyboard Shortcuts

### ISSUE THREAD

**Issue by aamancio** [warpdotdev/warp#1851]
### Describe the solution you'd like?

```text
I want to be able to CMD + Escape for a  Hotkey Window.
```

### Is your feature request related to a problem? Please describe.

```text
I'm just so used to doing it like that for years now with iTerm honestly, the only reason I haven't switched to Warp. I love the terminal, but I can't get over not having that Hotkey
```

### Additional context

<img width="434" alt="image" src="https://user-images.githubusercontent.com/79527088/191264446-aa8b0344-aecd-4984-a0b7-68fbc829dccc.png">

### How important is this feature to you?

5 (Can't work without it!)

**Comment by dannyneira** [comment]
Thanks for this feature request! Please +1 this request as it helps us gauge interest. We will post any updates here.

**Comment by dannyneira** [comment]
Include `CMD+BACKTICK` in the above request.

**Comment by egrefen** [comment]
+1

**Comment by yun-percy** [comment]
emmmm, it's just not working

v0.2022.11.14.14.55.stable_01

key:  ^+shift+alt+T.  or.  ctrl+alt+T. or cmd+ctrl+T

**Comment by victorkirov** [comment]
I don't think it has been implemented yet 😋

**Comment by eamigo86** [comment]
+1

**Comment by mguellsegarra** [comment]
+1

**Comment by markduan** [comment]
+1

**Comment by victorkirov** [comment]
+1

**Comment by rohatcan** [comment]
+1

**Comment by dannyneira** [comment]
Hey folks! In order to reduce notification spam, please add a :+1: to the original post at the top to signal that you want this feature, and then hit the subscribe button if you'd like to be notified when we have any updates.

**Comment by stephenmckinney** [comment]
> As a possible workaround, you can try using `OPTION-ESC`

I'm unable to use that also.

**Comment by EvgeniTroitskii** [comment]
Open warp one time per month, check this feature - close warp, still not use warp because key binding not works....

**Comment by daolou** [comment]
`CMD + .` also cannot work!

**Comment by Atoms** [comment]
I would like to have double Ctrl as hotkey, i'm using currently this as my iterm shortcut for global hotkey, and it's really not overriding any other shortcuts.

**Comment by dannyneira** [comment]
hey folks, quick update here. I found that some of the shortcuts listed `CMD-ESC`, `CMD-BACKTICK`, `CMD-TAB`, `CMD-PERIOD`, and `CMD-TILDE`, conflict with existing Mac system shortcuts, so to bind them to the Global Hotkey, you need to remove those bindings in mac system settings first, then you should be able to use the binding in Warp. 

Please try and let us know if that helps with the issue.

**Comment by mguellsegarra** [comment]
> hey folks, quick update here. I found that some of the shortcuts listed `CMD-ESC`, `CMD-BACKTICK`, `CMD-TAB`, `CMD-PERIOD`, and `CMD-TILDE`, conflict with [existing macOS system shortcuts](https://support.apple.com/en-us/HT201236), so to bind them to the Global Hotkey(or other Warp shortcuts), you need to remove those bindings in macOS system settings first, then you should be able to use the binding in Warp.
> 
> Please try and let us know if that helps with the issue.

I've just tested it and it's not working. I don't have any conflicting macOS system shortcuts with `CMD-ESC`. In fact, I can use `CMD-ESC` to control whatever I want in Mac OS settings (show/hide launchpad, etc.). I can also trigger iTerm 2 quake mode with this shortcut.

Sadly, no luck with Warp.

**Comment by obetomuniz** [comment]
I can set all of these shortcuts using iTerm 2, still don't understand why Warp doesn't allow us to override system shortcuts. I'm being blocked from test this term just because of this.

**Comment by samkrishna** [comment]
Yeah this is a mystery for me too. iTerm2 clearly lets me set Cmd-Esc as the hot key combo for Quake-style terminal window drop down. I've been using it without fail for going on 12-ish years now when I discovered it.

Right now, as it stands, I can't use Ctrl-Esc or Cmd-Esc for the hot key combo. Feels like a bug. I was able to get Opt-Esc to work, so there's that.

Also, I don't have Cmd-Esc mapped to anything at the System Settings level, so I'm going to go ahead and say that the feature instructions about un-setting the Cmd-Esc system-level key combo is wrong (aka has a documentation bug) unless there's something I'm missing.

**Comment by Jarvvski** [comment]
Shame - can't use Warp until this is fixed. iTerm 2 works fine with cmd + tilde/backtick

**Comment by PythonicNinja** [comment]
`cmd + .` works like charm on iterm2, not sure why you are blocking user preferences?

**Comment by SemenovSvyatoslav** [comment]
want to be able to CMD + Escape for a  Hotkey Window

**Comment by gfpacheco** [comment]
In my case I use just `` ` `` as hotkey on iTerm for years, can't live without it anymore as well

**Comment by ekaerik** [comment]
§ (tilde key on a Swedish keyboard) does not work as a hotkey either, would be awesome if that was added too.

**Comment by cgarrovillo** [comment]
What needs to be done to support this
